### DISCUSSION TITLE
Preview of the tracing feature in action?

### DISCUSSION THREAD

**Discussion by ringods** [concourse/concourse#6309] [Show and tell]
Hello,

Within the company I work for, I'm building up the case for Concourse as our single pane of glass regarding delivery flow. I found numerous screenshots around the net of Concourse pipelines themselves, coming from the Concourse web portal. But I would like to add to my proposal a screenshot of the [tracing feature](https://concourse-ci.org/tracing.html) of Concourse.

Is there anyone who has tracing enabled on their Concourse instance and can provide me with a screenshot of a full pipeline flow, coming from Jaeger tracing or Google StackDriver? You can blur sensitive information offcourse, for as long as the screenshot is still identifiable as coming from Concourse. 😄 

Thank you,

Ringo

**Comment by andy-paine** [comment]
I did a Lunch & Learn for EngineerBetter and I've got these screenshots of some example jobs - I haven't used it in anger yet though but I built a [Jaeger BOSH release](https://github.com/andy-paine/jaeger-boshrelease/tree/master/jobs) for users who wanted to try it.

![Jaeger UI view of a trace](https://i.imgur.com/6kBCJ1R.png)
![Grafana view of a trace](https://i.imgur.com/HyJdWiD.png)

**Reply by ringods** [reply]
Thank you very much @andy-paine!!
